reference/markdown/Markdown-DITA-syntax.md

```diff
@@ -0,0 +1,317 @@
+
+```markdown
+# Task {.task}
+
+Context
+
+1.  Command
+
+    Info.
+```
+
+```xml
+<task id="task">
+  <title>Task </title>
+  <taskbody>
+    <context>
+      <p>Context</p>
+    </context>
+    <steps>
+      <step>
+        <cmd>Command</cmd>
+        <info>
+          <p>Info.</p>
+        </info>
+      </step>
+    </steps>
+  </taskbody>
+</task>
+```
+
+The other way to use specialization types is by defining a [schema] for the document.
+
+## Sections
+
+The following class values in [header attributes] have a special meaning on heading levels other than 1:
+
+- `section`
+- `example`
+
+They are used to generate [`section`](https://docs.oasis-open.org/dita/v1.2/os/spec/langref/section.html) and [`example`](https://docs.oasis-open.org/dita/v1.2/os/spec/langref/example.html) elements:
+
+```markdown
+# Topic title
+
+## Section title {.section}
+
+## Example title {.example}
+```
+
+```xml
+<topic id="topic_title">
+  <title>Topic title</title>
+  <body>
+    <section>
+      <title>Section title</title>
+    </section>
+    <example>
+      <title>Example title</title>
+    </example>
+  </body>
+</topic>
+```
+
+## Tables
+
+Tables use the [MultiMarkdown] table extension format:
+
+<!-- Prevent Prettier from “fixing” cell span -->
+<!-- prettier-ignore-start -->
+```markdown
+| First Header | Second Header | Third Header |
+| ------------ | :-----------: | -----------: |
+| Content      |          _Long Cell_        ||
+| Content      |   **Cell**    |         Cell |
+```
+<!-- prettier-ignore-end -->
+
+Tables in Markdown DITA files are converted to the [OASIS exchange table model](https://www.oasis-open.org/specs/tm9901.html):
+
+```xml
+<table>
+  <tgroup cols="3">
+    <colspec colname="col1" colnum="1"/>
+    <colspec colname="col2" colnum="2"/>
+    <colspec colname="col3" colnum="3"/>
+    <thead>
+      <row>
+        <entry colname="col1">First Header</entry>
+        <entry align="center" colname="col2">Second Header</entry>
+        <entry align="right" colname="col3">Third Header</entry>
+      </row>
+    </thead>
+    <tbody>
+      <row>
+        <entry colname="col1">Content</entry>
+        <entry align="center" namest="col2" nameend="col3" colname="col2"><i>Long Cell</i></entry>
+      </row>
+      <row>
+        <entry colname="col1">Content</entry>
+        <entry align="center" colname="col2"><b>Cell</b></entry>
+        <entry align="right" colname="col3">Cell</entry>
+      </row>
+    </tbody>
+  </tgroup>
+</table>
+```
+
+Table cells may only contain inline content and column spans; block content and row spans are not supported in Markdown DITA.
+
+## Notes
+
+Notes can be written using the syntax defined in the Material for MkDocs [admonition extension](https://squidfunk.github.io/mkdocs-material/reference/admonitions/).
+
+<!-- Since 5.0 -->
+
+> **Note:** Requires DITA-OT 4.1 or newer.
+
+```markdown
+!!! note
+
+    Note content.
+```
+
+```xml
+<note>
+  <p>Note Content.</p>
```

Comment:
This syntax comes from the Material for MkDocs [admonition extension](https://squidfunk.github.io/mkdocs-material/reference/admonitions/), so I don't think we have much control over whether the output here contains paragraphs or not.

@jelovirt would know.

Reply:
I'm not suggesting we remove them. I expect they stay, and that's fine. What I was attempting to point out is that there could be an unexpected `<p>` in notes converted from Markdown that isn't present in XML. Should we call attention to it so that those who don't normally have `<p>` in their notes can plan to adjust their stylesheets?


```xml
<note>Note Content</note>
```

versus

```xml
<note>
  <p>Note Content</p>
</note>
```